### DISCUSSION TITLE
Need help with downloading open_agb_firm

### DISCUSSION THREAD

**Discussion by Nellyette** [profi200/open_agb_firm#207] [General]
Whenever I try to copy open_agb_firm in the payloads folder in luma, my computer gives out an error message of 0x8000FFFF:Catastropic Failure. I made sure to clear out the data in my 3ds SD card so it could be checked properly by h2testw, and it mentioned that the test finished without errors. I put my data back into my 3ds SD card, did the process of downloading open_agb_firm again, and it still showed the same error. 
![Screenshot 2024-09-20 234242](https://github.com/user-attachments/assets/af5fea25-2f0e-4114-8dd4-adb7a1c922db)

**Comment by profi200** [comment]
I have never seen an error like that and i don't know what's causing this. It's not really an issue with oaf so i'm moving this to the discussions tab.

Only advice i can give is to do basic troubleshooting. Try a different SD card. If it doesn't work try a different card reader and if that still doesn't work try a different PC.

**Comment by gabecj** [comment]
i had the same problem, so what i did was download the latest files of open_agb_firm on my phone, where i then unzipped it with zarchiver, although it will likely work with any file viewer. then i opened FTPD on the 3ds, and an app on my phone called "AndFTP" to connect the 3ds to the phone so i could directly download open_agb_firm.firm to the 3ds, bypassing the issue, and it worked (at least, as far as i know). might work with another app on the phone, though i havent done any other testing.

**Comment by Zyncosis** [comment]
I second this issue. Ive used 3 different machines and got the same error. Maybe something is up with the zip, not sure.
